tests/test_builders/conftest.py

```diff
@@ -21,6 +23,6 @@ def _parse(fname: Path) -> HTMLParser:
 
 
 @pytest.fixture(scope='package')
-def cached_etree_parse():
+def cached_etree_parse() -> Generator[Callable[[Path], Element], None, None]:
```

Comment:
same as above- i'm about 80% sure this can be `Iterable[Callable[[Path], Element]]`

Reply:
Same reply as above.